src/commands/npm/dependencies/hardcode.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { flags, FlagsConfig, SfdxCommand } from '@salesforce/command';
+import { Messages } from '@salesforce/core';
+import { Package } from '../../../package';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-release-management', 'npm.dependencies.hardcode');
+
+export default class Hardcode extends SfdxCommand {
+  public static readonly description = messages.getMessage('description');
+  public static readonly flagsConfig: FlagsConfig = {
+    dryrun: flags.boolean({
+      char: 'd',
+      default: false,
+      description: messages.getMessage('flags.dryrun'),
+    }),
+    release: flags.string({
+      char: 'r',
+      description: messages.getMessage('flags.release'),
+      default: 'latest',
+    }),
+  };
+
+  public async run(): Promise<Array<Record<string, string>>> {
+    const packageJson = await Package.create();
+    const pkg = packageJson.hardcodeDependencyVersions(this.flags.release);
+
+    if (this.flags.dryrun) {
+      process.emitWarning(`would print ${JSON.stringify(pkg['dependencies'], null, 4)} as the dependencies`);
```

Comment:
I gave an example of what I think the output should be. Then the warning here should just be `Running in --dryrun mode. No changes will be written to the package.json.`